### ISSUE TITLE
oci_image does not accept tar.xz

### ISSUE THREAD

**Issue by prestonvanloon** [bazel-contrib/rules_oci#249]
I read the example in the [aspect.dev migration guide from rules_docker](https://docs.aspect.build/guides/rules_oci_migration/). The example is as follows:
```python
genrule(
    name = "new_layer",
    srcs = [":deb"],
    outs = ["data.tar.xz"],
    cmd = "tar -xvf -c $@/ $(location :deb)",
)

oci_image(
    tars = [":new_layer"]
)
```
However, this fails as the tars attributes only accepts tar and tar.gz. 
https://github.com/bazel-contrib/rules_oci/blob/0218e5508d8f7b1ea64aaa65a9a96edff68e3f1e/oci/private/image.bzl#L52

A real world example with a workaround looks as follows:
```python
# WORKSPACE
http_file(
    name = "bash_amd64",
    urls = [
        "http://ftp.us.debian.org/debian/pool/main/b/bash/bash_5.1-2+deb11u1_amd64.deb",
    ],
    sha256 = "f702ef058e762d7208a9c83f6f6bbf02645533bfd615c54e8cdcce842cd57377",
)

http_file(
    name = "bash_arm64",
    urls = [
        "http://ftp.us.debian.org/debian/pool/main/b/bash/bash_5.1-2+deb11u1_arm64.deb",
    ],
    sha256 = "d7c7af5d86f43a885069408a89788f67f248e8124c682bb73936f33874e0611b",
)


# Some build file
genrule(
    name = "bash_tar",
    srcs = select({
        "@platforms//cpu:x86_64": ["@bash_amd64//file:file"],
        "@platforms//cpu:arm64": ["@bash_arm64//file:file"],
    }),
    outs = ["bash.tar"],
    cmd = "ar x $< && xz -d data.tar.xz -c >> $@",
    visibility = ["//visibility:public"],
)

oci_image(
    name = "image",
    tars = [
        ":bash_tar",
        ":binary_tar",
    ],
)
```

It would be nice to simply use this genrule, which doesn't un-xz the tarball, but I don't think that `tar -xvf` works for a deb file so I had to use `ar`.
```python
genrule(
    name = "new_layer",
    srcs = [":deb"],
    outs = ["data.tar.xz"],
    cmd = "ar x $< --output=$(@D)",
)

**Comment by thesayyn** [comment]
this can be fixed by simply adding `tar.xz`. we didn't add every possible extension on purpose to make sure we handle them correctly.

**Comment by thesayyn** [comment]
up for sending a PR adding it to this list?

**Comment by thesayyn** [comment]
fixed by #250
